controller.js

```diff
@@ -0,0 +1,16 @@
+const retreiveTopics = require('./models.js');
+
+function getTopics(req, res, next) {
+    const { query } = req
+    return retreiveTopics(query)
+
+    .then(({rows}) => {
+        res.status(200).send(rows)
+    })
+    .catch((err) => {
+        
+        next(err)
```

Comment:
this should just be catch(next), next(err) is used by your error handling middleware to pass it to the next handler in the chain. Look over the error handling notes again if you're unsure